mylyn.builds/org.eclipse.mylyn.hudson.core/META-INF/MANIFEST.MF

```diff
@@ -26,6 +24,8 @@ Bundle-Activator: org.eclipse.mylyn.internal.hudson.core.HudsonCorePlugin
 Import-Package: com.google.gson;version="[2.7.0,2.9.2)",
  com.sun.xml.bind;version="2.3.3",
  jakarta.activation;version="2.0.0",
```

Comment:
I switch to the 55_xlm branch and did a local build. When I use the local repository I get the following error.

```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Mylyn Builds Connector: Hudson/Jenkins 3.26.0.N20230320-1407 (org.eclipse.mylyn.hudson.feature.group 3.26.0.N20230320-1407)
  Missing requirement: Mylyn Hudson Connector 3.26.0.N20230320-1407 (org.eclipse.mylyn.hudson.core 3.26.0.N20230320-1407) requires 'java.package; jakarta.activation 2.0.0' but it could not be found
  Cannot satisfy dependency:
    From: Mylyn Builds Connector: Hudson/Jenkins 3.26.0.N20230320-1407 (org.eclipse.mylyn.hudson.feature.group 3.26.0.N20230320-1407)
    To: org.eclipse.equinox.p2.iu; org.eclipse.mylyn.hudson.core [3.26.0.N20230320-1407,3.26.0.N20230320-1407]
```

I need to delete the line 
 jakarta.activation;version="2.0.0",
from /org.eclipse.mylyn.hudson.core/META-INF/MANIFEST.MF to resolve this error.